### ISSUE TITLE
Can't activate projected system effect after deactivating it

### ISSUE THREAD

**Issue by blitzmann** [DarkFenX/Pyfa#355]
States seem to be getting confused

**Comment by blitzmann** [comment]
So, states for systems effects were never able to be changed. Before I made the changes to rigs to allow them to be turned on and off, system effects always remained "online". They were actually labeled as rigs on the pyfa side, presumably to keep pyfa from crying about the lack of slot on this "module".

So when I allowed rigs states to be changes, I also allowed system effects. I've fixed code the code so that the system effect states can now be properly toggled, but it leads to another problem: offline is the same as online.

A quick look at some system effect files shows `type = ("projected", "offline")` on pretty much all effects, so even if it's "offline" it would still run. I will have to continue looking into this. At the surface, I cannot see what we would have it as an offline effect other than the old assumption that if it was projected it was going to be applied anyways, without the thought of toggle. *shrug*
